### DISCUSSION TITLE
Subtitle for scedule and EventFahrplan

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1270] [Ideas]
Huhuu,

i was wondering if it would be possible to add a "Subtitle" for a Talk to display it in EventFahrplan and on the schedule.

or is there another way to display the abstract in the schedule?

Thank you very much!

**Comment by rixx** [comment]
There is currently no way to display additional information on the schedule – testing this with the abstract resulted in the decision that it was cluttering the schedule way too much.

A "subtitle" attribute has never been requested much, except by people looking to make pretalx more compatible with frab, so there haven't been any plans to add it so far.

**Reply by navlisilvan** [reply]
I think it is a pity that you decided against such a function.
Finally we found out how ot display the abstract in the Eventfahrplan and this was very useful. 

Unfortunately, the breaks of Pretalx are not exported in the XML and therefore not displayed in the Eventfahrplan, so we will have to create separate "Talks" for it next time.

**Reply by rixx** [reply]
I'm happy to merge a PR that adds breaks to the frab-compatible schedule export (which is community-maintained), **as long as** pretalx stays 100% compatible to the frab export format.
